public/components/GeoPlot.vue

```diff
@@ -712,6 +730,28 @@ export default Vue.extend({
     onFocusOut() {
       this.isImageDrilldown = false;
     },
+    mapToggle() {
+      this.isSatelliteView = !this.isSatelliteView;
+      this.map.remove(this.tileRenderer); // remove old tile renderer to destroy the buffers hold the previous tile set
+      this.tileRenderer = new lumo.TileLayer({
```

Comment:
From Line736 to 744 can be it's own method and be used in both `mapToggle()` and `createLumoMap()` methods.

Reply:
Added createMapLayers() to encompass this bit of code. Thanks!